docs/standard/native-interop/best-practices.md

```diff
@@ -251,6 +251,56 @@ finally
 }
 ```
```

Comment:
Also mention CLong as alternative option for `long` in the "Common Windows data types" table, and/or link from that table to this paragraph?